CHANGELOG.yaml

```diff
@@ -11,6 +11,16 @@ meta:
     - clients: the service API changed in a way that requires clients to be updated
 
 changes:
+  - title: Handle redirect response from DSS for processing in data-browser
+    issues:
+    - https://github.com/DataBiosphere/azul/issues/551
+    upgrade:
+    - deploy
+    - client
+    notes: When a download request is made to the service from the data-browser, the service obtains a redirect response
```

Comment:
Document the behavior in the docstring of the request handling method. @Ben-Wu did a good job doing the same for the manifest endpoint in https://github.com/DataBiosphere/azul/pull/558/files#diff-a3dec4f6b35a08e7cb29086cb8766c3aR554 (plus my suggestions). Only put a reference to that docstring in these notes.